styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java

```diff
@@ -221,6 +221,7 @@ private void setUpConfig() {
     System.setProperty("styx.test.namespace", testNamespace);
     System.setProperty("styx.api.port", String.valueOf(styxApiPort));
     System.setProperty("styx.scheduler.port", String.valueOf(styxSchedulerPort));
+    ConfigFactory.invalidateCaches();
```

Comment:
Accidentally we run flyte test case first and then styx native in this case and that surfaces this issue. More specifically changing of property `styx.test.namespace` won't be reflected because typesafe config caches.